config/application.properties

```diff
@@ -1,3 +1,4 @@
+## Spring Boot Application Properties should go in this file.
 spring.main.sources = com.capgemini.brahma.Application
```

Comment:
can you look at this file and env.properties at the HEAD of master in camel-boot? I moved these from file to file and added a few more comments.